reload/dpl-platform-poc
# Experimenting with ingress cert manager task modules

#### What does this PR do?
Restructures task files scripts and manifests

#### How should this be manually tested?
Verify that task commands can be run both from "main" task file and in individual task files.
